redirects.txt

```diff
@@ -200,11 +200,10 @@ crm/optimize/setup.rst applications/general/voip/asterisk.rst                 #
 
 general/odoo_basics/add_user.rst applications/general/odoo_basics/users.rst                               # add_user -> users
 applications/general/in_app_purchase/in_app_purchase.rst applications/general/in_app_purchase.rst         # in_app_purchase/in_app_purchase -> in_app_purchase
-applications/general/tags/take_tags.rst applications/general/tags.rst                                     # tags/take_tags -> tags
 applications/general/multi_companies/manage_multi_companies.rst applications/general/multi_companies.rst  # multi_companies/manage_multi_companies -> multi_companies
 applications/general/payment_acquirers/payment_acquirers.rst applications/general/payment_acquirers.rst   # payment_acquirers/payment_acquirers -> payment_acquirers
-applications/general/unsplash/unsplash_access_key.rst applications/general/unsplash.rst                   # unsplash/unsplash_access_key -> unsplash
-applications/general/unsplash/unsplash_application_id.rst applications/general/unsplash.rst               # unsplash/unsplash_application_id -> unsplash
+applications/general/unsplash/unsplash_access_key.rst applications/websites/website/optimize/unsplash.rst
+applications/general/unsplash/unsplash_application_id.rst applications/websites/website/optimize/unsplash.rst
```

Comment:
New rules should be added to ensure the coherence of links.
If someone bookmarked `applications/general/unsplash`, he won't be redirected to the new page location.

Reply:
thanks, applied.